### PR TITLE
Add throw expressions

### PR DESCRIPTION
**Pull request by jridgewell**
Stage 2 proposal: https://github.com/tc39/proposal-throw-expressions.

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
"throw" should also be added here:
https://github.com/babel/babylon/blob/master/src/types.js#L419-L426

**Comment by jridgewell** [comment]
Done.

File: src/types.js [hunk 1]
```diff
@@ -423,7 +423,8 @@ export type UnaryOperator =
   | "~"
   | "typeof"
   | "void"
-  | "delete";
+  | "delete",
```
Comment (nicolo-ribaudo) [inline src/types.js]: I think the comma here is illegal

Comment (jridgewell) [reply src/types.js]: Whoops.

**Pull request merged**